src/app/options-modal/options-modal.component.ts

```diff
@@ -117,9 +117,13 @@ export class OptionsModalComponent {
   }
 
   easyModeChange(event: Event){
-    if (!(event.target instanceof HTMLInputElement)) return;
-    this.gameStateService.easyModeEver = true;
-    this.characterService.characterState.easyMode = event.target.checked;
+    event.preventDefault();
+    if (!this.characterService.characterState.easyMode && confirm("This will enable easy mode and mark your save permanently. Are you sure?")){
```

Comment:
I'd use the easyModeEver flag for this check. If they're turning if back on after turning it on and off it's not a big deal.